pygmt/src/grdcontour.py

```diff
@@ -28,7 +34,7 @@
     t="transparency",
 )
 @kwargs_to_strings(
-    R="sequence", L="sequence", A="sequence_plus", c="sequence_comma", p="sequence"
+    R="sequence", L="sequence", A="sequence", c="sequence_comma", p="sequence"
```

Comment:
Should be remove `A="sequence"` because we need to process this argument manually.
```suggestion
    R="sequence", L="sequence", c="sequence_comma", p="sequence"
```